File: Python/FizzBuzz.py
for i in range(100):
  multiple_of_three = i % 3 == 0
  multiple_of_five = i % 5 == 0

  result = ''

  if (multiple_of_three):
    result += 'Fizz'
  
  if (multiple_of_five):
    result += 'Buzz'

  print(result if len(result) > 0 else i)

File: Kotlin/FizzBuzz.kt
fun fizzBuzz(start: Int, end: Int) {
    for (k in start..end) {
        val multipleOfThree = k % 3 == 0
        val multipleOfFive = k % 5 == 0

        val resultBuilder = StringBuilder()
        
        if (multipleOfThree) {
            resultBuilder.append("Fizz")
        }
        
        if (multipleOfFive) {
            resultBuilder.append("Buzz")
        }

        if (resultBuilder.isEmpty()) continue

        println(resultBuilder.toString())
    }
}

fun main() {
    fizzBuzz(1, 100)
}